File: cpp/1188.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

float sum(float m[], int s, int e)
{
    float sum = 0;

    for (s; s < e; s++) {
        sum += m[s];
    }

    return sum;
}

int main()
{
    int left = 133, right = 143, i;
    float m[144], result = 0;
    char op;
    cin >> op;

    for (i = 0; i < 144; i++) {
        cin >> m[i];
    }

    while (left > 88) {
        result += sum(m, left, right);
        left -= 11;
        right -= 13;
    }

    if (op == 'S') {
        printf("%.1f\n", result);
    } else {
        printf("%.1f\n", result / 30);
    }

    return 0;
}

File: py/1188.py
operation = input()
matrix = []

i = 0
while i < 144:
    matrix.append(float(input()))
    i += 1

result = 0
left = 133
right = 143

while True:
    result += sum(matrix[left : right : 1])
    
    left -= 11
    right -= 13

    if left < 88:
        break

if operation == 'S':
    print(f'{(result):.1f}')

else:
    print(f'{(result / 30):.1f}')